README.md

```diff
@@ -71,3 +71,74 @@ docker run --env-file .env --add-host=host.docker.internal:host-gateway --detach
 
 And you're done! This mini-tutorial doesn't support the leaderboard, though. You also may wish to set up a simple
 docker compose file. [Join the discord](https://valk.sh/discord) to learn more!
+
+
+# Using Docker Compose
+You can copy the variables below, or grab them from [here](<https://github.com/PBOwner/xpd/blob/prod/docker-compose.yml>)
+
+## Docker Compose File:
+
+```bash
```

Comment:
could we embed this from somewhere? Duplicating it seems like a bad idea.

Reply:
> could we embed this from somewhere? Duplicating it seems like a bad idea.

You can always modify it! I set it so that you can as needed. I'm not sure i "duplicated" as i just modified files and added a docker-compose.yml file

Reply:
no, as in we have the compose.yml but we also have the same content in readme

Reply:
Oh, yea i can fix that really quickly. Give me a second